library/src/options.cxx

```diff
@@ -106,57 +106,79 @@ class options::internals
-  this->Internals->init("emissive-factor", std::vector<double>{ 1., 1., 1. });
-  this->Internals->init("geometry-only", false);
-  this->Internals->init("line-width", 1.0);
-  this->Internals->init("metallic", 0.0);
-  this->Internals->init("normal-scale", 1.0);
-  this->Internals->init("opacity", 1.0);
-  this->Internals->init("point-size", 10.0);
-  this->Internals->init("progress", false);
-  this->Internals->init("roughness", 0.3);
-  this->Internals->init("texture-base-color", std::string());
-  this->Internals->init("texture-emissive", std::string());
-  this->Internals->init("texture-material", std::string());
-  this->Internals->init("texture-normal", std::string());
-  this->Internals->init("up", std::string("+Y"));
-
-  // Rendering/Dynamic
-  this->Internals->init("axis", false);
-  this->Internals->init("background-color", std::vector<double>{ 0.2, 0.2, 0.2 });
-  this->Internals->init("bar", false);
-  this->Internals->init("blur-background", false);
-  this->Internals->init("cells", false);
-  this->Internals->init("colormap",
+  // Loader
+  this->Internals->init("loader.animation.index", 0);
+  this->Internals->init("loader.camera.index", -1);
+  this->Internals->init("loader.up-direction", std::string("+Y"));
+
+  this->Internals->init("loader.geometry-only", false);
+  this->Internals->init("loader.show-progress", false);
+
+  this->Internals->init("loader.default.line-width", 1.0);
+  this->Internals->init("loader.default.point-size", 10.0);
+
+  this->Internals->init("loader.default.color.rgb", std::vector<double>{ 1., 1., 1. });
+  this->Internals->init("loader.default.color.opacity", 1.0);
+  this->Internals->init("loader.default.color.texture", std::string());
+
+  this->Internals->init("loader.default.emissive.factor", std::vector<double>{ 1., 1., 1. });
+  this->Internals->init("loader.default.emissive.texture", std::string());
+
+  this->Internals->init("loader.default.normal.texture", std::string());
+  this->Internals->init("loader.default.normal.scale", 1.0);
+
+  this->Internals->init("loader.default.material.metallic", 0.0);
+  this->Internals->init("loader.default.material.roughness", 0.3);
+  this->Internals->init("loader.default.material.texture", std::string());
+
+  // Window
+  this->Internals->init("window.background.color", std::vector<double>{ 0.2, 0.2, 0.2 });
+  this->Internals->init(
+    "window.background.hdri", std::string()); // XXX This overrides window.background.color
+  this->Internals->init("window.background.blur", false);
+
+  this->Internals->init("window.camera.azimuth-angle", 0.0);
+  this->Internals->init("window.camera.elevation-angle", 0.0);
+  this->Internals->init("window.camera.focal-point", std::vector<double>());
+  this->Internals->init("window.camera.position", std::vector<double>());
+  this->Internals->init("window.camera.view-angle", 0.0);
+  this->Internals->init("window.camera.view-up", std::vector<double>());
+
+  this->Internals->init("window.coloring.cells", false);
```

Comment:
I think those are related to `loader`, it's just sciviz material parameters.

Reply:
it can be changed dynamically in the window, it is related to rendering.

Reply:
will be reworked